src/main/docker/jenkins/Dockerfile

```diff
@@ -1,54 +1,21 @@
 FROM jenkins/jenkins:lts
 
-LABEL description="Jenkins with maven, python3.6 and gcc libraries pre-installed for Artemis"
+LABEL description="Jenkins with plugins pre-installed for Artemis"
 
 USER root
 
 RUN apt update
```

Comment:
As far as I remember it is best practice to replace apt with apt-get in docker files as apt is used as end user CLI. :)